plugins/output/flight/flight.go

```diff
@@ -0,0 +1,156 @@
+package flight
+
+import (
+	"context"
+	_ "embed"
+	"log"
+	"net"
+
+	"github.com/apache/arrow/go/v9/arrow"
+	"github.com/apache/arrow/go/v9/arrow/array"
+	"github.com/apache/arrow/go/v9/arrow/flight"
+	"github.com/apache/arrow/go/v9/arrow/ipc"
+	"github.com/apache/arrow/go/v9/arrow/memory"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+// DO NOT REMOVE THE NEXT TWO LINES! This is required to embed the sampleConfig data.
+//
+//go:embed sample.conf
+var sampleConfig string
+
+func (*Flight) SampleConfig() string {
+	return sampleConfig
+}
+
+type Flight struct {
```

Comment:
Added a doc comment explaining what the struct is. Explaining each individual member seems excessive, I haven't seen any other plugin do it this way.

Reply:
If this is generally not done in the Go community it may be fine to wait and only add the remaining doc comments until `flight.go` becomes complex enough that it would be hard to comprehend without. What do you thing @CGodiksen and @chrthomsen?

Reply:
For now I think it is fine to leave it as it is. I imagine this struct might become more complex when supporting different schemas so this can be reconsidered then.

Reply:
I agree, let's close this one.